.cpp

```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+Int main()
+{
+    Int m1,m2,m3,m4,m5,per;
+    Printf("enter marks in five subjects");
+    Scanf("%d%d%d%d%d",&m1,&m2,&m3,&m4,&m5);
+
+     Per=(m1+m2+m3+m4+m5)/500*100;
+     
+    If(per>=60)
+          Printf("first division\n");
+     
+    If((per>=50)&&(per<50))
```

Comment:
Can you please use || and ! as well in this code so that it could become more informative.

Reply:
Ok sir I'll try to do again and will add some more stuff here.